snowplow_tracker/test/unit/test_tracker.py

```diff
@@ -223,12 +223,12 @@ def test_track(self) -> None:
 
         t = Tracker("namespace", [e1, e2, e3])
 
-        p = Payload({"test": "track"})
+        p = Payload({"eid":"event_id"})
         t.track(p)
```

Comment:
Whenever possible, every PR should have a unit test verifying the functionality. You can either check here that the track call returns the expected event ID or make a new test case for it.